src/main/java/com/epam/wl/dao/UserDAO.java

```diff
@@ -31,12 +33,14 @@
             "SELECT * FROM user WHERE email=? AND passwordhash=?";
     private static final String GET_USER_SCRIPT_By_NAME_LASTRNAME_SCRIPT =
             "SELECT * FROM user WHERE name=? AND lastname=?";
+    private static final String IS_LIBRARIAN_QUERY = "SELECT * FROM user WHERE email=? AND passwordhash=?";
```

Comment:
You don't need password to check user's role. Also try to find smith about EXISTS sql keyword, you can use to return boolean value from a request

Reply:
I agree that a password is a redundant parameter. As for the query type, according to the signature of the executeQuery method, I must pass some ResultSet as a second parameter. So, if I get a boolean value from a request, I couldn't call this method